Source/BusinessMomentsAdaptorImage/dataschema.ts

```diff
@@ -0,0 +1,25 @@
+import mongoose from 'mongoose';
+
+const uri: string = 'mongodb://127.0.0.1:27017/local';
+
+mongoose.connect(uri, (err: any) => {
+    if (err) {
+        console.log(err.message);
+    } else {
+        console.log('Successfully Connected to MongoDB!');
+    }
+});
+
+export const DataSchema = new mongoose.Schema({
```

Comment:
I don't think we can use schema for the payload. If the payload changes we'll break the schema. Try to rename some of the properties - like publisher. What happens? My bet is that the publisher field will not be stored - it will be ignored.

Reply:
I agree and I was thinking of that before. But I was not sure how to make a generic schema to handle any type of payload. And yes, in case you have a different payload (the keys), the key will not be stored. Needs to be fixed